units/systemd-udevd.service.in

```diff
@@ -8,7 +8,7 @@
 #  (at your option) any later version.
 
 [Unit]
-Description=Rule-based manager for device events and files
+Description=Rule-Based Manager for Device Events and Files
```

Comment:
This was also changed in some other PR... I think there'll be a conflict, because it should be "Rule-based" not "Rule-Based".